BlueShift-iOS-SDK/BlueShiftUserInfo.m

```diff
@@ -29,9 +33,10 @@ - (NSDictionary *)toDictionary {
     if (self.email) {
         [sharedUserInfoMutableDictionary setObject:self.email forKey:@"email"];
     } else {
-        #ifdef DEBUG
-            NSLog(@"\n\n BlueShiftWarning: Email not set for BlueShiftUserInfo \n");
-        #endif
+        if (!isEmailIdNotSetInfoDisplayed) {
+            [BlueshiftLog logInfo:@"EmailId is not set for BlueShiftUserInfo. Please set email id." withDetails:nil methodName:nil];
+            isEmailIdNotSetInfoDisplayed = YES;
```

Comment:
I assume this is to print this error only once?

Reply:
Yes, this way it will print it only for once.